Add sponsorship badge (#46)
fastruby/skunk: README.md

## Contributing

Bug reports and pull requests are welcome on GitHub at https://github.com/fastruby/skunk/issues.

> ## FastRuby.io

Reviewer: ```suggestion
## Sponsorship
```

> ![fastruby](https://github.com/fastruby/skunk/raw/master/fastruby-logo.png)

Reviewer: ```suggestion
![FastRuby.io | Rails Upgrade Services](https://github.com/fastruby/skunk/raw/master/fastruby-logo.png)
```

Reviewer: @bronzdoc Any thoughts about this? Our brand is "FastRuby.io" or "FastRuby.io | Rails Upgrade Services" -- source: https://fastruby.github.io/styleguide/

Author: @etagwerker I like "FastRuby.io | Rails Upgrade Services"

> `skunk` is maintained and funded by FastRuby.io, inc. The names and logos for FastRuby.io are trademarks of FastRuby.io, inc.

Reviewer: ```suggestion
`skunk` is maintained and funded by [FastRuby.io](https://fastruby.io). The names and logos for FastRuby.io are trademarks of The Lean Software Boutique LLC.
```
